frontend/src/layout/navigation-3000/sidepanel/panels/SidePanelSupport.tsx

```diff
@@ -100,6 +100,28 @@ const SupportFormBlock = ({ onCancel }: { onCancel: () => void }): JSX.Element =
 
     return (
         <Section title="Email an engineer">
+            <SupportForm />
+            <LemonButton
+                form="support-modal-form"
+                htmlType="submit"
+                type="primary"
+                data-attr="submit"
+                fullWidth
+                center
+                className="mt-4"
+            >
+                Submit
+            </LemonButton>
+            <LemonButton
+                form="support-modal-form"
+                type="secondary"
+                onClick={onCancel}
+                fullWidth
+                center
+                className="mt-2"
+            >
+                Cancel
+            </LemonButton>
             <div className="grid grid-cols-2 border rounded [&_>*]:px-2 [&_>*]:py-0.5 mb-4 bg-bg-light">
```

Comment:
definitely need to add some top-padding here!
```suggestion
            <div className="grid grid-cols-2 border rounded [&_>*]:px-2 [&_>*]:py-0.5 mb-4 bg-bg-light pt-4">
```

Reply:
Ah good catch, thanks!  I tried adding that `pt-4` on my local, but it has no visible effect.  So I went old-school and tried `<br />` just after the 'cancel' button.  Looks good, so I'll make that edit here and request another review....

![CleanShot 2024-08-26 at 17 27 16@2x](https://github.com/user-attachments/assets/6f9309c4-f93d-4e64-9d01-9a2adb1e425f)

Reply:
Hm, I wonder if that's because of the `[&_>*]:py-0.5` class, which should only impact the children elements but maybe it causes a conflict?

I'm not a fan of the `<br />`, it should really be done with CSS. You can try `mt-4` on this element, or `mb-4` on the Cancel button instead.

Reply:
Oh, yes. Much better, thanks @raquelmsmith !

<img width="524" alt="image" src="https://github.com/user-attachments/assets/36871b56-3ab8-4a9f-8179-87d6fd9a74df">
<img width="541" alt="image" src="https://github.com/user-attachments/assets/7e687b7c-5f4b-4b1f-a291-7a92fee29ab2">